Client/Script/BulkShiftOverlay.js

```diff
@@ -422,13 +440,28 @@ class BulkShiftOverlay {
             TableElements.shortTimeColumn('New End')
         );
 
+        const markerTypeSelected = this.markerType();
         const sortedMarkers = BulkActionCommon.sortMarkerList(shiftResult.allMarkers, shiftResult.episodeData);
+
+        this.#markersWithoutRows.length = 0; 
+
         for (let i = 0; i < sortedMarkers.length; ++i) {
             const checkGroup = [];
             const eInfo = shiftResult.episodeData[sortedMarkers[i].parentId];
-            checkGroup.push(sortedMarkers[i]);
+
+            
+            if (markerTypeSelected == 'both' || sortedMarkers[i].markerType == markerTypeSelected)  { // If the marker type is selected, prep it for row addition
+                checkGroup.push(sortedMarkers[i]);
+            }
+            else this.#markersWithoutRows.push(sortedMarkers[i].id); // If not then store it for the getIgnored table later
```

Comment:
Nit style: All control flow statements should have braces to be consistent with the rest of the codebase.

```suggestion
            else {
                this.#markersWithoutRows.push(sortedMarkers[i].id); // If not then store it for the getIgnored table later
            }
```